src/Domain/Creators/BinaryCompactObject.cpp

```diff
@@ -102,6 +104,11 @@ BinaryCompactObject::BinaryCompactObject(
   } else {
     projective_scale_factor_ = 1.0;
   }
+
```

Comment:
Just a nitpick: the extra line here isn't consistent with there no being extra lines between the `if` statements above